libs/api-interfaces/src/lib/aggregate-views/coscrad-data-types/coscrad-model-schema.ts

```diff
@@ -0,0 +1,8 @@
+// Do we really want to default to any?
+
+import { CoscradPropertyTypeDefinition } from './coscrad-property-type-definition';
+
+// eslint-disable-next-line
+export type ICoscradModelSchema<T extends Record<string, unknown> = any> = {
```

Comment:
We may want to look at where the default is being used and see if it is necessary.